test/integration_test.py

```diff
@@ -23,6 +23,8 @@
 # system run all envs, but each env and each system is run at least once.
 # For each system we select a random environment to run.
 # Then for each environment we select a random system to run.
+anakin_config_path = "../mava/configs/default/anakin"
+
 ppo_systems = [
     "ppo.anakin.ff_ippo",
```

Comment:
Here I'm thinking we could do something like the below when we have all the systems for the sebulba arch else we do them separately:
```suggestion
arch="anakin"
anakin_config_path = f"../mava/configs/default/{arch}"
ppo_systems = [
    f"ppo.{arch}.ff_ippo",
```